index.bs

```diff
@@ -126,20 +133,20 @@ the [=construct a Sensor object=] abstract operation.
 ### Accelerometer.x ### {#accelerometer-x}
 
 The {{Accelerometer/x!!attribute}} attribute of the {{Accelerometer}}
-interface represents the [=acceleration=] along x-axis.
-This attribute returns [=map/value=] for [=latest reading=]["x"] [=map/entry=].
+interface returns the result of invoking [=get value from latest reading=] with
+<emu-val>this</emu-val> and "x" as arguments.
```

Comment:
I'd add:

>It represent the [=acceleration=] along x-axis.

And similarly for the rest.

Otherwise, the spec does not define the semantics explicitly, only indirectly in:

>whose keys are "x", "y", "z" and whose values contain device’s acceleration about the corresponding axes

This is also to follow the convention of:

>The foo attribute returns bar. It represents baz.

Reply:
done